KISSmetrics-iOS-SDK.podspec.json

```diff
@@ -2,14 +2,14 @@
   "name": "KISSmetrics-iOS-SDK",
   "version": "2.3.1",
   "summary": "iOS SDK for KISSmetrics",
-  "homepage": "http://www.kissmetrics.com",
-  "documentation_url": "http://support.kissmetrics.com/article/show/ios-sdk-v2",
+  "homepage": "https://www.kissmetrics.io",
+  "documentation_url": "http://support.kissmetrics.io/article/show/ios-sdk-v2",
```

Comment:
This links have to be re-mapped unfortunately: https://support.kissmetrics.io/reference#ios-v2

Reply:
Will also want to use `https`

Reply:
Fixed in 6cb22e1